test/unit/api/fragment.js

```diff
@@ -119,4 +119,24 @@ describe('api/fragment', function () {
       expect(frag.childNodes[1].tagName).to.equal('TD');
     });
   });
+
+  describe('node', function () {
+    it('should work with element nodes', function () {
+      const node = document.createElement('div');
+      const frag = skate.fragment(node);
+      expect(frag.childNodes[0]).to.equal(node);
```

Comment:
Ended up writing a separate test in 4b0a923 since that test is working with a plain `<div>`.